### PR TITLE
More thanos crap

### PR DESCRIPTION
**Pull request by ghost**
:cl:
add: Added a new spell for thanos -- Tank Armor. It allows him to "focus" his armor on either ballistics, melee, or lasers. Crew, use a VARIETY of weapons!
balance: However, as a consequence, his normal armor is nerfed a bit.
imageadd: Icons for shockwave and bullcharge
fix: You can no longer insta-crit thanos or the syndie stone holder by bullcharging into each other.
balance: Gauntlet regen is now at 3/second, and syndie stone regen is at 5/s.
balance: Gauntlet AND syndie stone regen are halved while in space.
balance: If you leave Thanos in crit, to die, without bothering to finish him off... It may come back to bite you in the ass.
tweak: Slightly buffed THANOS CAR damage.
/:cl:

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
I misread this as thanos carp and was mildly dissapointed

**Comment by ExpAntique** [comment]
+1

**Comment by Majick1** [comment]
+1, but I thought thanos could already snap while in crit unless you're talking about something else. Would help to flesh out the PRs if you put more than just the changelog notes

**Comment by cb583** [comment]
I like this better than removing uranium

**Comment by ghost** [comment]
do keep in mind you can shoot even in melee and if you wanna insta crit someone use high caliber uranium bullet

**Comment by Manly-Dwarf** [comment]
So Thanos' general armor is worsened, but now he has a Vaccinator.

File: hippiestation/code/modules/gauntlet/ganymede.dm [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
 	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
-	armor = list("melee" = 50, "bullet" = 65, "laser" = 65, "energy" = 45, "bomb" = 100, "bio" = 30, "rad" = 30, "fire" = 70, "acid" = 30)
+	armor = list("melee" = 35, "bullet" = 45, "laser" = 45, "energy" = 45, "bomb" = 100, "bio" = 30, "rad" = 30, "fire" = 70, "acid" = 30)
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/gauntlet/ganymede.dm]: This armor is comparable to the gem-encrusted hardsuit and is worse than the syndicate elite hardsuit.
https://github.com/HippieStation/HippieStation/blob/f36040b814f46c69ecb66b06cdd8af59e47f3f69/code/modules/clothing/spacesuits/hardsuit.dm#L360
Finna crit Thanos in two shotgun blasts or one fireaxe.

**Comment by ExpAntique** [comment]
Bruh why not make his bullet and armour resistance like 90 or some shit. Fuck it, he’s Thanos, dude could tank legit anything

**Comment by GuyonBroadway** [comment]
Thanos needs to be somewhat beatable, otherwise it's no fun for everyone else.

As nice as it would be to make thanos mostly invincible and hope he roleplays out a Cinematic fight vs the station.... if you find a way to the parallel universe where THAT happens do let me know.

**Comment by mikolaj007** [comment]
+1 i guess

**Comment by ghost** [comment]
the whole point of the tank armor spell is to make it so you can't just kill him with shotgun spam.

He ain't dying to shotguns? Get out the lasers. He ain't dying to lasers? Get out the shotguns.

**Comment by Manly-Dwarf** [comment]
I like this better than the uranium nerf but it still sounds potentially scary, to make the already tanky Thanos even tankier. Although you did at least nerf his healing in cold/on fire.

**Comment by AnthonyJNgo** [comment]
This is much better than the uranium shell removal, +1 after reviewing.

**Pull request merged**